cmd/debugger/main.go

```diff
@@ -66,12 +66,18 @@ func handleWinChangeData(ptmx *os.File, data []byte) error {
 }
 
 func populateShellHistory(cmd string) error {
-	f, err := os.Create("/root/.bash_history")
-	if err != nil {
-		return err
+	for _, f := range []string{
+		"/root/.ash_history",
+		"/root/.bash_history",
+	} {
+
+		f, err := os.Create(f)
+		if err != nil {
+			return err
```

Comment:
Wonder if we should `continue`?

Reply:
good point, how about I switch it over to using multierrors?

Reply:
sounds good